packages/tradershub/src/features/cfd/screens/MT5AccountTypeCard/MT5AccountTypeCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Text } from '@deriv/quill-design';
+
+type TProps = {
+    description: string;
+    icon: JSX.Element;
+    isSelected: boolean;
+    onClick: () => void;
+    title: string;
+};
+
+const MT5AccountTypeCard: React.FC<TProps> = ({ description, icon, isSelected, onClick, title }) => {
+    return (
+        <div
+            className={
+                isSelected
+                    ? 'rounded-800 border-brand-blue bg-system-light-primary-background cursor-pointer border-solid border-sm'
+                    : 'cursor-pointer'
+            }
+            onClick={onClick}
+            onKeyDown={e => {
+                if (e.key === 'Enter' || e.key === ' ') {
+                    onClick();
+                }
+            }}
+            role='button'
+            tabIndex={0}
+        >
+            <div className='flex w-[26.4rem] h-[25rem] items-start'>
```

Comment:
I'm not sure whether in `@deriv/quill-design` they correct `rem` values, 1px = 10rem. It will be safer if we stick to `px` as of now don't you think 🤔

Reply:
Im sticking to what we have from wallets. Since that is the reference we have for now

Reply:
@thisyahlen-deriv as I checked with @michio quill doesn't understand rem, so we need to change them into px